packages/flutter_tools/lib/src/cache.dart

```diff
@@ -62,10 +62,11 @@ class Cache {
         locked = true;
       } on FileSystemException {
         if (!printed) {
-          printStatus('Waiting to be able to obtain lock of Flutter binary artifacts directory...');
+          printTrace('Waiting to be able to obtain lock of Flutter binary artifacts directory: ${_lock.path}');
+          printStatus('Waiting for another flutter command to finish...');
```

Comment:
Fair point.

How about "Waiting for another flutter command to release its lock of the Flutter binary artifacts directory..."?
Or if that's too long, "Waiting for another flutter command to release the startup lock..."?

I mean, in practice, it might not be another flutter command (it's just a BSD flock), but that's probably a safe enough assumption for most users.